### PR TITLE
Namespace api to /api/v1/

### PR DESCRIPTION
**Pull request by Zeko369**
- [] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [] My pull request targets the `master` branch of Chapter.

Closes #XXXXX

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -4,11 +4,12 @@ import express from 'express';
 import morgan from 'morgan';
 import { responseErrorHandler } from 'express-response-errors';
 
-import exampleRouter from 'server/routes/exampleRoutes/exampleRouter';
-import chapterRouter from 'server/routes/chapter';
-import eventsRouter from 'server/routes/events';
+// import exampleRouter from 'server/routes/exampleRoutes/exampleRouter';
```
Comment (vaibhavsingh97) [inline server/index.ts]: I think comments can be removed

File: server/index.ts [hunk 2]
```diff
@@ -28,9 +29,11 @@ nextjs.nextApp.prepare().then(async () => {
   app.use(express.json());
   app.use(express.static('public'));
 
-  app.use(exampleRouter);
-  app.use(chapterRouter);
-  app.use(eventsRouter);
+  // app.use(exampleRouter);
+  // app.use(chapterRouter);
+  // app.use(eventsRouter);
```
Comment (vaibhavsingh97) [inline server/index.ts]: same as above

File: server/index.ts (hunk 1 above)
Comment (Zeko369) [reply server/index.ts]: Forgot about that

**Comment by vaibhavsingh97** [comment]
waiting for another approval

**Pull request merged**